From add8d4ffe1804d77399d3e62aefd4c12667edaa9 Mon Sep 17 00:00:00 2001
From: Surface Duo Builder <bariccatti@gmail.com>
Date: Sun, 19 Apr 2026 19:49:21 +0000
Subject: [PATCH] Remove Samsung HIDL dependencies from powermanager and
 surfaceflinger

vendor/interfaces no longer ships Samsung HIDL stubs (stripped for
Surface Duo). Remove vendor.samsung.hardware.miscpower@2.0 deps.
---
 services/powermanager/Android.bp                     | 2 --
 services/surfaceflinger/CompositionEngine/Android.bp | 1 -
 2 files changed, 3 deletions(-)

diff --git a/services/powermanager/Android.bp b/services/powermanager/Android.bp
index ef01e13..78f8f7b 100644
--- a/services/powermanager/Android.bp
+++ b/services/powermanager/Android.bp
@@ -42,7 +42,6 @@ cc_library_shared {
         "android.hardware.power@1.1",
         "android.hardware.power@1.2",
         "android.hardware.power@1.3",
-        "vendor.samsung.hardware.miscpower@2.0",
     ],
 
     export_shared_lib_headers: [
@@ -50,7 +49,6 @@ cc_library_shared {
         "android.hardware.power@1.1",
         "android.hardware.power@1.2",
         "android.hardware.power@1.3",
-        "vendor.samsung.hardware.miscpower@2.0",
     ],
 
     whole_static_libs: [
diff --git a/services/surfaceflinger/CompositionEngine/Android.bp b/services/surfaceflinger/CompositionEngine/Android.bp
index 5d0c8ed..259e86b 100644
--- a/services/surfaceflinger/CompositionEngine/Android.bp
+++ b/services/surfaceflinger/CompositionEngine/Android.bp
@@ -39,7 +39,6 @@ cc_defaults {
         "libSurfaceFlingerProp",
         "libui",
         "libutils",
-        "vendor.samsung.hardware.miscpower@2.0",
     ],
     static_libs: [
         "libguiflags",
-- 
2.34.1

